src/app/components/input/input.component.ts

```diff
@@ -0,0 +1,43 @@
+import {Component, EventEmitter, Output} from '@angular/core';
+
+type InputLengthI = { title?: number, body?: number }
+
+@Component({
+  selector: 'app-input',
+  templateUrl: './input.component.html',
+  styleUrls: ['./input.component.scss']
+})
+export class InputComponent {
+  showFirst = true;
+  showDropdownMenu = false;
+
+  toggleDropdownMenu() {
+    this.showDropdownMenu = !this.showDropdownMenu;
+  }
+
+  toggleDivs() {
+    this.showFirst = !this.showFirst;
+  }
+
+  title: string = '';
+  text: string = '';
+
+  @Output() saveNote = new EventEmitter<{ title: string, text: string }>();
```

Comment:
same as above we should have the @Input and @Output at the start of the component instead of in the middle of component